src/modality-specific-files/microscopy.md

```diff
@@ -102,7 +102,8 @@ Consistent with other data types in BIDS, the session entity is optional.
 
 The [`sample-<label>`](../appendices/entities.md#sample) entity is REQUIRED for
 Microscopy data and is used to distinguish between different samples from the same subject.
-The label MUST be unique per subject and is RECOMMENDED to be unique throughout the dataset.
+Contrary to other labels, the `sample-<label>` MUST be unique per subject
+and is RECOMMENDED to be unique throughout the dataset.
```

Comment:
I do not see why I am to be disallowed to use `sample-1`, `sample-2` and so on across my subjects.
It seems that the intent here to demand some UUIDs,  but I do not see really why here and why only for `sample`s.

Moreover, as we ATM have mandatory `sub-` leading folder anyways, I do not understand really what "unique per subject" means really.